java/examples/deployment-modes/run_examples.sh

```
@@ -0,0 +1,207 @@
+USAGE
+}
+
+# Locate the latest non-classifier-test cudf-*.jar in the Maven repo.
+# Accept any cuda* classifier (e.g. cudf-26.08.0-SNAPSHOT-cuda12.jar) but
+# skip *-tests.jar / *-sources.jar / *-javadoc.jar.
+locate_cudf_jar() {
+  if [[ ! -d "$CUDF_GROUP_DIR" ]]; then
+    err "cudf Maven artifacts not found at $CUDF_GROUP_DIR. Build cudf first (mvn install)."
+  fi
+  local jar
+  jar=$(ls -1t "$CUDF_GROUP_DIR"/*/cudf-*.jar 2>/dev/null \
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)